### PR TITLE
remember that easing previewer plugin? yeah, I finally uploaded it

### PR DESCRIPTION
**Pull request by ThaCuber**
[in case you don't get what I mean](https://www.youtube.com/watch?v=y6_Prq8hRGE)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -148,6 +148,13 @@
       "id": "dragdropselected",
       "mod_version": "3"
     },
+    {
+      "description": "Adds a popup that displays the curve of Penner-styled easing functions.",
+      "version": "1.0",
+      "remote": "https://github.com/ThaCuber/lite-xl-easingpreview",
```
Comment (Guldoman) [inline manifest.json:154]: You must specify a commit hash like this:
```suggestion
      "remote": "https://github.com/ThaCuber/lite-xl-easingpreview:ea349d12a9570264408dc1567af660b8d4568d3d",
```

Might also be worth it to add a `name` field.

Comment (ThaCuber) [reply manifest.json:154]: whi though (the commit hash)

Comment (Guldoman) [reply manifest.json]: To pin the version to the commit (and as a way to avoid malicious changes).

Comment (ThaCuber) [reply manifest.json]: also, can I write "Easing Previewer" for the `name` field?

Comment (Guldoman) [reply manifest.json]: Yep.

**Pull request merged**